docs/src/index.md

```diff
@@ -268,6 +268,178 @@ df2 = @eachrow df begin
 end
 ```
 
+## Row-wise transformations with `@byrow`
+
+DataFrames provides the function-wrapper `ByRow`. `ByRow(f)(x, y)`
+is roughly equivalent to `f.(x, y)`, with a few exceptions discussed below. 
+DataFramesMeta allows for users to construct expressions using `ByRow` 
+function wrapper with the syntax `@byrow`. 
+
+```julia
+@transform(df, y = @byrow :x == 1 ? "true" : "false)
+```
+
+becomes
+
+```
+transform(df, :x => ByRow(x -> x == 1 ? "true", "false") => :y)
+```
+
+!!! note 
+    Unlike `@.`, `@byrow` is not a "real" macro and cannot be used outside of 
```

Comment:
the question is what are the parsing rules for `@byrow`? does it take exactly one expression that follows it?

Reply:
I included a more detailed discussion in updated docs. Hopefully it's clear.